.github/workflows/ci.yml

```diff
@@ -100,7 +105,7 @@ jobs:
       - name: Cache dependencies
         run: ./ci/before_cache.sh
 
-  unixish:
+  posix:
```

Comment:
What's wrong with "unixish"?

Reply:
I recall someone mentioning that posix was the more appropriate term. It's not an important distinction to me, I'll revert it.

Reply:
Yep; "Posix" is a well-defined term, while "unixish" is not.

Reply:
Ah, I first thought "posix" is too general, but it turns out to be the other way around.

Reply:
OK, re-adding the posix then :)